src/runrms/executor/_rms_executor.py

```diff
@@ -22,85 +21,45 @@ class RMSExecutor(ABC):
 
     def __init__(self, config: InteractiveRMSConfig | FMRMSConfig) -> None:
         self._config = config
+        self._exec_env = self._init_exec_env()
 
-        # We want to append to or carry forward the following env vars
-        pre_env = {}
-        pre_env["PYTHONPATH"] = os.environ.get("PYTHONPATH", None)
```

Comment:
It was here to help with testing but if kept will eventually be a source of mysterious misbehavior. It is not used in a default Komodo.